CONTRIBUTING.md

```diff
@@ -12,10 +12,20 @@ The main project fork lives here in Github:
 
 * [https://github.com/G-Research/armada](https://github.com/G-Research/armada)
 
+To work with [Armada website](https://armadaproject.io/), please checkout this branch:
```

Comment:
This isn't universally true, for some docs, they need to be updated in master branch (such as this one). I'd suggest linking directly to https://github.com/G-Research/armada/blob/gh-pages/README.txt and https://github.com/G-Research/armada/blob/master/docs/docs-readme.md to help explain the relationship.